### DISCUSSION TITLE
Can you combine a modal wizard with stickyModalFooter?

### DISCUSSION THREAD

**Discussion by henninghorn** [filamentphp/filament#12729] [Help]
### Package

Actions

### Package Version

v3.2.76

### How can we help you?

I'm using a modal to display a wizard using this approach: https://filamentphp.com/docs/3.x/actions/modals#using-a-wizard-as-a-modal-form

I have a step in the wizard with a lot of content, which therefore pushes the "next"-button out of the viewport, so that you have to scroll down to see it.

Normally with a modal with a lot of content, you could use the `stickyModalFooter()` method (https://filamentphp.com/docs/3.x/actions/modals#making-the-modal-footer-sticky), in order to always display the buttons at the bottom.

However, this doesn't seem to work with a modal that has a wizard.
You can use the `stickyModalHeader()` to keep your header displayed while scrolling, but you can't use the `stickyModalFooter()`.

What is the best workaround for this?

I'm concidering using a regular form within the modal, with a wizard component, where I hide the wizard controls.
And then add my own controls for the modal's footer. However, this approach seems a little hacky to me.

**Comment by w2ca** [comment]
@leandrocfe I'm using the wizard in a modal, and I'm having the same problem as him, is there a solution for this?

**Comment by henninghorn** [comment]
@Willy-Bueno 

My current solution is to use the modal blade component directly and then use the `sticker-footer` option:

```blade
<x-filament::modal slide-over sticky-header sticky-footer id="booking-widget-modal" width="6xl">
  <div>
    <livewire:dynamic-component :is="$this->getStepComponent()" :key="$this->currentStep" />
  </div>
  <x-slot name="footer">
    <!-- Buttons here -->
  </x-slot>
</x-filament::modal>
```

It works for now :)

However, it seems to be a bug, that the modal doesn't support the sticky footer option.
I may look into it myself in september and make a PR if I find a fix.
